packages/synapse-react-client/src/lib/assets/ButtonWithIcon.tsx

```diff
@@ -1,28 +0,0 @@
-import * as React from 'react'
```

Comment:
Unused, MUI makes this easy for us with the `startIcon` prop on the `Button` component